meta/main.yml

```diff
@@ -0,0 +1,21 @@
+galaxy_info:
+  role_name: foreman_modules
+  author: The Foreman
+  description: Deploy Foreman Ansible Modules as a Role
+  issue_tracker_url: https://github.com/ericsysmin/ansible-role-foreman-modules/issues
+  license: GPLv3
+  min_ansible_version: 2.0.0
```

Comment:
I think it might be a later version, at least for some modules. I'd rather not have this if we can help it?

Reply:
Will make it a later version like the latest release